From 873c4eaf89d37a4fab623782189e947130e0c1a7 Mon Sep 17 00:00:00 2001
From: Leah Rowe <leah@libreboot.org>
Date: Mon, 5 Oct 2026 00:51:03 +0100
Subject: [PATCH 1/1] don't download or reset ipxe revs

lbmk handles it as a submodule and we copy it. this
is for edk2, also handled by lbmk (built by coreboot).

Signed-off-by: Leah Rowe <leah@libreboot.org>
---
 payloads/external/iPXE/Makefile | 17 ++++-------------
 1 file changed, 4 insertions(+), 13 deletions(-)

diff --git a/payloads/external/iPXE/Makefile b/payloads/external/iPXE/Makefile
index a9fb43f846..54940ac3e5 100644
--- a/payloads/external/iPXE/Makefile
+++ b/payloads/external/iPXE/Makefile
@@ -22,23 +22,14 @@ endif
 all: build
 
 $(project_dir):
-	echo "    Cloning $(project_name) from Git"
-	git clone $(project_git_repo) $(project_dir)
+	rm -Rf ipxe
+	cp -a ipxe_src ipxe
 
 fetch: $(project_dir)
-	cd $(project_dir); \
-		git show $(TAG-y) >/dev/null 2>&1 ; \
-		if [ $$? -ne 0 ] || [ "$(TAG-y)" = "origin/master" ]; then \
-			echo "    Fetching new commits from the $(project_name) repo"; \
-			git fetch; \
-		fi
+	:
 
 checkout: fetch
-	echo "    Checking out $(project_name) revision $(TAG-y)"
-	cd  $(project_dir); \
-		git checkout master; \
-		git branch -D coreboot 2>/dev/null; \
-		git checkout -b coreboot $(TAG-y)
+	:
 
 config: checkout
 ifeq ($(CONSOLE_SERIAL),yy)
-- 
2.47.3

